src/System.Net.Http.WinHttpHandler/tests/FunctionalTests/WinHttpHandlerTest.cs

```diff
@@ -38,25 +38,18 @@ public void SendAsync_SimpleGet_Success()
         }
 
         [Fact]
-        public async Task GetAsync_Cancel_CancellationTokenPropagates()
```

Comment:
There are a bunch of HTTP tests in System.Net.Http/tests/FunctionalTests that really exercise the functionality.  Since System.Net.Http contains the WinHttpHandler (Windows ) and CurlHandler (*Nix), most of the tests are there.

I'm in the process of reorganizing the HTTP tests.  The intent for System.Net.Http.WinHttpHandler tests is to test specific functionality such as properties that aren't present in the main HttpClientHandler but are present in the WinHttpHandler.

This particular test will probably get moved to HttpClientHandlerTest.cs in the System.Net.Http tests at some point in the future.